helm/cilium/templates/hubble-relay/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```

Comment:
We are not planning to use PSPs in openstack but the API is still registered with the server. Can you please add `psp.enabled` (and disabled by default) in values.yaml?